### DISCUSSION TITLE
Ability select custom Server ID

### DISCUSSION THREAD

**Discussion by yodaphone** [alexjustesen/speedtest-tracker#491] [Ideas]
When you select the server ID, i think it shows the nearest servers available & doesnt show all possible servers. For e.g. i would like to test the speed to a server in another country and i have the server ID is it possible to test against that?

**Comment by alexjustesen** [comment]
Correct, it's based on "your" location which is how the Speedtest CLI returns the server list. There is a discussion somewhere here where we started chatting about logging all the server IDs but haven't expanded on that idea yet.
